pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -4433,7 +4433,7 @@ func (r *ClickHouseReader) GetLogAttributeValues(ctx context.Context, req *v3.Fi
 
 }
 
-func readRow(vars []interface{}, columnNames []string) ([]string, map[string]string, []map[string]string, v3.Point) {
+func readRow(vars []interface{}, columnNames []string, countOfNumberCols int) ([]string, map[string]string, []map[string]string, v3.Point) {
```

Comment:
If the number of `Number` cols is just 1 in the result, then there is no ambiguity so it will be interpreted as the `value` column.